docker-compose.yaml

```
@@ -75,15 +72,13 @@ services:
       - ./datasource.yml:/etc/grafana/provisioning/datasource.yml
     env_file:
       - ./config.monitoring
-    networks:
-      example-network:
-        ipv4_address: 172.16.238.12
 
-networks:
-  example-network:
-    name: example-network
-    driver: bridge
-    ipam:
-      driver: default
-      config:
-        - subnet: 172.16.238.0/24
\ No newline at end of file
+  mongodb-exporter:
+    image: ssheehy/mongodb-exporter:latest
+    ports:
+      - 9216:9216
+    environment:
+      MONGODB_URI: 'mongodb://mongodb_exporter:s3cr3tpassw0rd@mongo-db:27017/main?sslmode=disable'
+    links:
+      - mongo-db
+      - prometheus
```

```suggestion
      - prometheus
      
```

Newline at end of file